larray/inout/common.py

```diff
@@ -39,16 +41,21 @@ class FileHandler:
 
     Parameters
     ----------
-    fname : str or Path
+    fname : str or Path, optional
```

Comment:
It is not really optional at this point, as it has no default. It accepts None, which is not the same thing as being optional. Maybe it should be made optional, I don't know.